wallet-new/src/Cardano/Wallet/Orphans/Aeson.hs

```diff
@@ -27,13 +27,16 @@ instance FromJSON BackupPhrase where
 instance ToJSON CFilePath where
   toJSON (CFilePath c) = toJSON c
 
+-- FIXME: This couple of instances is not complementary,
+-- can't write roundtrip test for them.
+-- It doesn't matter though, because we will hardly need to return password
```

Comment:
Yes, this is something I want to fix for V1. I my opinion Aeson instances should always roundtrip 😉

Reply:
Then we can rewrite `ToJSON` so that for empty passphrase it produces `null`.
But whether do we need `instance ToJSON PassPhrase` at all?

Reply:
I don't understand why don't these instances roundtrip, honestly. `Arbitrary` instance for `PassPhrase` doesn't allow for passphrases which are not 0 or 32 bytes long. It seems like that if these instances don't roundtrip, then one of them is buggy =\